packages/mongodb-chatbot-server/src/routes/conversations/createConversation.ts

```diff
@@ -53,7 +57,10 @@ export function makeCreateConversationRoute({
         res,
         createConversationCustomData
       );
-      const conversationInDb = await conversations.create({ customData });
+      const conversationInDb = await conversations.create({
```

Comment:
Should it be an error to create a conversation with no system prompt? Do we need to modify the conversations service / add tests for this case?

Reply:
hmm. let's just make the `systemPrompt` required. i was thinking about flexibility when i was doing the initial implementation. but w/ some more thought, it's probably easier and footgun avoid-y to make required. and like who wouldn't want a system prompt?!?

also i realize that we need to update the docs and the quick start w/ this change. will include in the PR as well.

Reply:
Seems reasonable for the route. I'm okay if we don't want to update this at the conversations service level (i.e. require at least one initial message) though I'm not sure we'll ever need the flexibility there either.

As a follow on to making `systemPrompt` required in this route we can clean up the line that assumes it's optional, i.e.

```ts
initialMessages: systemPrompt ? [systemPrompt] : [],
```
can just be
```ts
initialMessages: [systemPrompt],
```

Reply:
Careful, `[undefined].length === 1`

Reply:
Not sure I follow - per the latest changes `systemPrompt` is required and won't be `undefined`

Reply:
Gotcha